### PR TITLE
Donor footer tweaks

### PR DESCRIPTION
**Pull request by clareconstantine**
Incorporate feedback to improve the donor footer - adjust fonts, margins, prevent the background image from tiling, etc.

**Teacher wide:**
Screen Shot 2019-09-18 at 11 17 17 AM

**Teacher narrow:**
Screen Shot 2019-09-18 at 11 19 48 AM

**Student wide:**
Screen Shot 2019-09-18 at 11 17 42 AM

**Student narrow:**
Screen Shot 2019-09-18 at 11 20 11 AM

### CODE REVIEW

**Comment by dju90** [comment]
Looks great!

File: dashboard/app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -3118,23 +3118,28 @@ h2.danger {
...
   padding-top: 30px;
-  background-size: cover;
-  min-height: 340px;
+  padding-bottom: 36px;
 
   h1 {
     color: $white;
     margin-bottom: 24px;
+    margin-top: 0;
+    font-family: "Gotham 5r";
```
Comment (breville) [inline dashboard/app/assets/stylesheets/application.scss]: We usually also provide a second fallback `sans-serif` font.

**Comment by breville** [comment]
nit: do we need `1-` and `2-` in the image filenames?  It seems it might be a bit cleaner without them.

**Comment by breville** [comment]
nit: When on certain narrow widths, the text does go all the way to the edge of the page when we'd normally want a bit of left & right margin/padding.  

I just realised (and should have noticed when testing your PR before) that /home is actually not built as a responsive page, leading to the interesting case that the donor footer is the one piece that actually is actually responsive.  @Erin007 might have contrary thoughts, but I actually don't mind seeing it act responsively here, even when nothing else (except the header) does.

I do not have contrary thoughts.  If the header and footer are the only responsive elements that seems just fine and a step the right direction if we want to responsify the whole page in the future.

**Comment by clareconstantine** [comment]
> nit: do we need `1-` and `2-` in the image filenames? It seems it might be a bit cleaner without them.

@breville I am using them to enforce the order of the logos - The list of files is sorted alphabetically and this allows us to control the order.

**Pull request merged**